### ISSUE TITLE
Backtracking

### ISSUE THREAD

**Issue by noob-26** [Algo-Phantoms/Algo-Tree#42]
Can I work on Backtracking problems like Sudoku solver, N Queens and finding all subsets?

**Comment by Vish-07** [comment]
Heyy!! I would love to do bactracking problems of rat maze, color filling algorithm, n queens problem under mentorship of GSSOC'21. Will you please assign this task to me?

**Comment by rudrakshi99** [comment]
@noob-26 mention the language in which you want to add the code.

**Comment by noob-26** [comment]
C++

**Comment by noob-26** [comment]
I want to contribute to this repo as part of GSSoC 2021. So should I make the PR after 8th March, when the coding period starts?

**Comment by rudrakshi99** [comment]
@noob-26 Yes so that your point will be counted in GSSoC.

**Comment by noob-26** [comment]
Thanks a lot for the help.

**Comment by mgncodes** [comment]
Hello! I'd like to add solutions to some famous backtracking problems in Python 3 as a part of GSSoC '21. Please assign this issue.
@noob-26

**Comment by mgncodes** [comment]
Hi, I have made a PR with the solution for N-queens problem for this issue #350 . Kindly do the needful. 
@rudrakshi99

**Comment by rudrakshi99** [comment]
> Can I work on Backtracking problems like Sudoku solver, N Queens and finding all subsets?

Please make separate issues for each algorithm and you will work on a sudoku solver in this issue.

**Comment by noob-26** [comment]
@rudrakshi99 Please can you review my code for the Sudoku solver problem 
Pull request #735

**Comment by raksha009** [comment]
> Hello! I'd like to add solutions to some famous backtracking problems in Python 3 as a part of GSSoC '21. It would be very helpful if you assign this issue to me.
> @rudrakshi99
> @raksha009

Heyy, this issue wasn't assigned to you. Plz create a pr only after the issue is assigned. 
Also, if you wanna work on this problem. Plz create a different issue addressing to only 1 problem. Closing your pr for now.

**Comment by mgncodes** [comment]
> > Hello! I'd like to add solutions to some famous backtracking problems in Python 3 as a part of GSSoC '21. It would be very helpful if you assign this issue to me.
> > @rudrakshi99
> > @raksha009
> 
> Heyy, this issue wasn't assigned to you. Plz create a pr only after the issue is assigned.
> Also, if you wanna work on this problem. Plz create a different issue addressing to only 1 problem. Closing your pr for now.
Hi, Sorry for the mistake. I have created a separate issue #1320 . Kindly do the needful. 
@raksha009

**Comment by raksha009** [comment]
> > > Hello! I'd like to add solutions to some famous backtracking problems in Python 3 as a part of GSSoC '21. It would be very helpful if you assign this issue to me.
> > > @rudrakshi99
> > > @raksha009
> > 
> > 
> > Heyy, this issue wasn't assigned to you. Plz create a pr only after the issue is assigned.
> > Also, if you wanna work on this problem. Plz create a different issue addressing to only 1 problem. Closing your pr for now.
> 
> Hi, Sorry for the mistake. I have created a separate issue #1320 . Kindly do the needful.
> @raksha009

Heyy, the issue is already present and assigned. Please look for other issues :)

**Comment by yashmita** [comment]
I would like to work on this issue in Java. Kindly assign it to me @raksha009.

**Comment by shahina-bano** [comment]
I would like to work on this issue in python. Kindly assign it to me @plazzy99

**Comment by raksha009** [comment]
> Hello! I'd like to add solutions to some famous backtracking problems in Python 3 as a part of GSSoC '21. It would be very helpful if you assign this issue to me.
> @rudrakshi99
> @raksha009

Go ahead with sudoku solver  in python

**Comment by shahina-bano** [comment]
I would like to work on this issue in python. Kindly assign it to me @raksha009.
